contrib/scripts/umeemainnet_fork.sh

```diff
@@ -0,0 +1,181 @@
+FORK_DIR="${FORK_DIR:-$CWD}"
+CHAIN_DIR="${CHAIN_DIR:-$FORK_DIR/node-data}"
+LOG_LEVEL="${LOG_LEVEL:-debug}"
+BLOCK_TIME="${BLOCK_TIME:-6}"
+UPGRADE_TITLE="${UPGRADE_TITLE:-"v1.0-v3.0"}"
+MAINNET_EXPORTED_GENESIS_URL="${MAINNET_EXPORTED_GENESIS_URL:-"https://storage.googleapis.com/umeedropzone/jul-28-umee-1-export.json.gz"}"
+UMEED_BIN_CURRENT="${UMEED_BIN_CURRENT:-$FORK_DIR/../../build/umeed}"
+UMEEMAINNET_GENESIS_PATH="${UMEEMAINNET_GENESIS_PATH:-$CWD/tinkered_genesis.json}"
+NODE_PRIV_KEY="${NODE_PRIV_KEY:-$FORK_DIR/priv_validator_key.json}"
+SEC_AWAIT_NODE_START="${SEC_AWAIT_NODE_START:-80}"
+
+# Loads another sources
+. $CWD/blocks.sh
+
+nodeHome="$CHAIN_DIR/$CHAIN_ID"
+home="--home $nodeHome"
+
+hdir="$CHAIN_DIR/$CHAIN_ID"
+
+# Folders for nodes
+nodeDir="$hdir/node"
+
+# Home flag for folder
+nodeHome="--home $nodeDir"
+
+# Config directories for nodes
+nodeCfgDir="$nodeDir/config"
+
+# Config files for nodes
+nodeCfg="$nodeCfgDir/config.toml"
+
+# App config files for nodes
+nodeApp="$nodeCfgDir/app.toml"
+
+# Node private key to sign blocks
+nodePrivateKeyPath=$nodeDir/config/priv_validator_key.json
+
+# Common flags
+kbt="--keyring-backend test"
+cid="--chain-id $CHAIN_ID"
+
+ADMIN_KEY="admin"
+NODE_MNEMONIC="festival dumb luxury boss forum clip scatter moral ribbon language rib unable burden agree burden misery suspect find crucial seat canvas endless bring lobster"
+NEWLINE=$'\n'
+
+genesisConfigPath="config/genesis.json"
+
+# Checks for private_key file
+if test -f "$NODE_PRIV_KEY"; then
+  echo "$NODE_PRIV_KEY exists."
+else
+  echo "$NODE_PRIV_KEY does not exists, exiting..."
+  exit 1
+fi
+
+# Checks for the tikered genesis file
+if [ ! -f "$UMEEMAINNET_GENESIS_PATH" ]; then
+  echo "$UMEEMAINNET_GENESIS_PATH doesn't exist"
+  EXPORTED_GENESIS_UNPROCESSED=$CWD/umeemainnet.genesis.json
+
+  if [ ! -f "$EXPORTED_GENESIS_UNPROCESSED" ]; then
+
+    EXPORTED_GENESIS_UNZIPED=$CWD/umeemainnet.genesis.json.gz
+
+    if [ ! -f $EXPORTED_GENESIS_UNZIPED ]; then
+      echo "$EXPORTED_GENESIS_UNZIPED doesn't exist, we need to curl it"
+      curl $MAINNET_EXPORTED_GENESIS_URL > $EXPORTED_GENESIS_UNZIPED
+    fi
+
+    echo "$EXPORTED_GENESIS_UNPROCESSED doesn't exist, we need to unpack"
+    gunzip -k $EXPORTED_GENESIS_UNZIPED
+  fi
+
+  echo "$EXPORTED_GENESIS_UNPROCESSED exists and ready to be processed"
+  EXPORTED_GENESIS_UNPROCESSED_PATH=$EXPORTED_GENESIS_UNPROCESSED COSMOS_GENESIS_TINKERER_SCRIPT=umeemainnet-fork.py EXPORTED_GENESIS_PROCESSED_PATH=$UMEEMAINNET_GENESIS_PATH $CWD/process_genesis.sh
+fi
+
+echo Remove everything from the $CHAIN_DIR
+rm -rf $CHAIN_DIR
+
+echo Start the chain basic config
+$UMEED_BIN_MAINNET $nodeHome $cid init $ADMIN_KEY
+
+echo Create node admin wallet
+yes "$NODE_MNEMONIC$NEWLINE" | $UMEED_BIN_MAINNET $nodeHome keys add $ADMIN_KEY $kbt --recover
+NODE_ADDR="$($UMEED_BIN_MAINNET $nodeHome keys show $ADMIN_KEY -a $kbt)"
+
+cp $NODE_PRIV_KEY $nodePrivateKeyPath
+
+echo Coping addr is $NODE_ADDR
```

Comment:
mention of `coping`

Reply:
removed at https://github.com/umee-network/umee/pull/1334/commits/95d603c61263d0ae41b342f5c1ff129ade21d1e5